testpackage/tests/Flowthrough_damr/Flowthrough_damr.cfg

```diff
@@ -0,0 +1,113 @@
+ParticlePopulations = proton
+
+project = Flowthrough
+propagate_field = 1
+propagate_vlasov_acceleration = 1
+propagate_vlasov_translation = 1
+dynamic_timestep = 1
+
+[proton_properties]
+mass = 1
+mass_units = PROTON
+charge = 1
+
+[AMR]
+max_spatial_level = 2
+adapt_refinement = 1
+use_alpha1 = 1
+use_alpha2 = 1
+refine_cadence = 2
+
+[gridbuilder]
+x_length = 16
+y_length = 8
+z_length = 8
+x_min = -8e7
+x_max = 8e7
+y_min = -4e7
+y_max = 4e7
+z_min = -4e7
+z_max = 4e7
+t_max = 182.0
```

Comment:
The pulse propagates about 22% of the simulation domain here. I would recommend propagating it about 80% and adding output files at 20,40,60, and 80% extents.

Reply:
Adjusted per recommendation.